src/store/index-level.ts

```diff
@@ -37,75 +33,124 @@ export class IndexLevel {
     return this.db.close();
   }
 
-  async put(id: string, entry: Entry, options?: IndexLevelOptions): Promise<void> {
-    entry = flatten(entry) as Entry;
-
-    const ops: LevelWrapperBatchOperation<string>[] = [ ];
-    const prefixes: string[] = [ ];
-    for (const property in entry) {
-      const value = entry[property];
-
-      const prefix = this.join(property, this.encodeValue(value));
-      ops.push({ type: 'put', key: this.join(prefix, id), value: id });
-      prefixes.push(prefix);
+  /**
+   * Adds indexes for a specific data/object/content.
+   * @param id ID of the data/object/content being indexed.
+   */
+  async put(
+    id: string,
```

Comment:
another variable naming idea. `id` feels vague to me, even though you define it in the comment above this method. What do you think of changing `id` -> `dataId` throughout this file?

Reply:
Glad you suggest it. I wasn't sure `dataId` sounds "okay" or not, but sounds like you like it. Changing now.